includes/class-wc-stripe-woocommerce-helper.php

```diff
@@ -0,0 +1,22 @@
+<?php
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+use Automattic\WooCommerce\Utilities\OrderUtil;
+
+/**
+ * Provides static methods as helpers for WooCommerce-related features.
+ *
+ * @since 8.6.0
+ */
+class WC_Stripe_WooCommerce_Helper {
+	/**
+	 * Checks if the custom orders table is enabled.
+	 *
+	 * @return bool Whether the custom orders table is enabled.
+	 */
+	public static function is_custom_orders_table_enabled() {
+		return class_exists( 'Automattic\WooCommerce\Utilities\OrderUtil' ) && OrderUtil::custom_orders_table_usage_is_enabled();
+	}
```

Comment:
I'm not against adding this new class but I wanted to double check incase you hadn't considered it, but there's a pretty similar class already called `WC_Stripe_Woo_Compat_Utils`.

Its described as WooCommerce version helpers but I think it could be used for stuff like this. What do you think?

Reply:
It makes sense! I didn't realize the context of that class before by its name (`compat` did not ring any bells for me until I opened it). Moved this method there 👍